notebooks/dev_prod_sites_api_explore.ipynb

```diff
@@ -0,0 +1,259 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Explore the functions of the Quartz Solar Site API using Dev and Prod API"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For loading credentials from .env file\n",
+    "import os\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "# For data manipulation\n",
+    "import pandas as pd\n",
+    "\n",
+    "# For API requests\n",
+    "import requests\n",
+    "import json\n",
+    "\n",
+    "# For plotting\n",
+    "import plotly.express as px"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "environment = \"dev\"  # or \"prod\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The access token can be inputed in two way:\n",
+    "1. Load the access token in from a .env file where the token is saved as a string variable valled \"ACCESS_TOKEN\"\n",
+    "    In this .env file have two access tokens called \"PROD_ACCESS_TOKEN\" and \"DEV_ACCESS_TOKEN\"\n",
+    "2. Making a request to the API using login crededntials to recieve the access token"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# A function to specify the environment and load the correct urls and credentials\n",
+    "def set_environment_variables(environment):\n",
+    "    load_dotenv()  # Ensure environment variables are loaded\n",
+    "    if environment == \"prod\":\n",
+    "        base_url = \"https://api-site.quartz.solar\"\n",
+    "        client_id = \"QLMdXCCHMS9Zl5W2tDG3otpdcY7GeEnJ\"\n",
+    "        domain = \"nowcasting-pro.eu.auth0.com\"\n",
+    "        access_token = os.getenv('PROD_ACCESS_TOKEN')\n",
+    "        if access_token is None:\n",
+    "            raise ValueError(\"PROD_ACCESS_TOKEN not found in environment variables\")\n",
+    "    elif environment == \"dev\":\n",
+    "        base_url = \"http://uk-development-sites-api.eu-west-1.elasticbeanstalk.com\"\n",
+    "        client_id = \"QLMdXCCHMS9Zl5W2tDG3otpdcY7GeEnJ\"\n",
```

Comment:
should these client ids be different, should they maybe even be in .env file?

Reply:
actually i think its fine they are there

Reply:
Yes, so its the same client_id but a different base_url. I'll just set it once.